ssntp/server.go

```diff
@@ -248,6 +249,15 @@ func (server *Server) Serve(config *Config, ntf ServerNotifier) error {
 	transport := config.transport()
 	uri = config.URI
 
+	payload, err := configuration.ConfigurationBlob(config.ConfigURI)
+
+	if err != nil {
+		server.log.Errorf("Error loading configuration: %s", err)
+		return err
```

Comment:
Please do not return an error for now, until at least we have deployed configuration files in our test clusters.
Keep the Errorf() log for people to know they're missing the configuration file, but be more verbose:
""Error loading configuration data from %s: %s - You may have not installed your configuration file yet", err, config.ConfigURI)